packages/@aws-cdk-testing/framework-integ/test/aws-synthetics/test/integ.canary-auto-delete-lambda.ts

```diff
@@ -54,8 +54,21 @@ class TestStack extends Stack {
 
 const app = new App();
 
-new IntegTest(app, 'cdk-integ-synthetics-canary-auto-delete', {
+const integTest = new IntegTest(app, 'cdk-integ-synthetics-canary-auto-delete', {
   testCases: [new TestStack(app, 'cdk-synthetics-canary-auto-delete')],
   diffAssets: true,
   stackUpdateWorkflow: false, // will error because this stack has a cr that deletes its own resources
 });
+
+// Assertion for Canary creation
+const canaryAssertion = integTest.assertions.awsApiCall('Synthetics', 'describeCanaries', {
+  MaxResults: 10, // Adjust this if needed
+});
+
+canaryAssertion.expect(ExpectedResult.objectLike({
```

Comment:
Let's remove this assertion and just leave this one as is. I feel like the only way we could reasonably assert that the canary was deleted is by calling `getCanary` for the deleted canary, but this will result in an assertion stack failure due to a `ResourceNotFoundException`.

Reply:
Thank you @colifran for cleaning it up.